### PR TITLE
poc monorepo migration

### PR DESCRIPTION
**Pull request by dhayab**
> **Warning**
> 
> Do not merge!

This PR lists steps taken to prepare algoliasearch-helper-js for its integration into the instantsearch monorepo.

We've been able to successfully release a version with ShipJS in a copy of this repository (with package name changed).

FX-2279

### CODE REVIEW

**Comment by Haroenv** [comment]
maybe the bundlesize file can be copied (and helper added) so that CI will actually run

**Comment by dhayab** [comment]
@Haroenv I added a commit that fixes / mocks a few things to allow the monorepo pipeline to run.

File: packages/algoliasearch-helper/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+How to contribute?
```
Comment (Haroenv) [inline packages/algoliasearch-helper/CONTRIBUTING.md:1]: we can probably remove this

File: packages/algoliasearch-helper/src/SearchParameters/RefinementList.js [hunk 2]
```diff
@@ -130,7 +130,7 @@ var lib = {
         if (facetList.length !== values.length) {
           hasChanged = true;
         }
-        // eslint-disable-next-line no-param-reassign
+
```
Comment (Haroenv) [inline packages/algoliasearch-helper/src/SearchParameters/RefinementList.js:133]: why was this one no longer reported?

**Review by Haroenv — APPROVED** [review]
👌 changes look solid to me

File: packages/algoliasearch-helper/CONTRIBUTING.md (hunk 1 above)
Comment (dhayab) [reply packages/algoliasearch-helper/CONTRIBUTING.md:1]: You're right. I added a "cleanup" section that explicitly mentions this file (and others) in the notes to follow during import process (https://algolia.atlassian.net/browse/FX-2048).

File: packages/algoliasearch-helper/src/SearchParameters/RefinementList.js (hunk 2 above)
Comment (dhayab) [reply packages/algoliasearch-helper/src/SearchParameters/RefinementList.js:133]: It is disabled in the monorepo [.eslintrc.js](https://github.com/algolia/instantsearch/blob/master/.eslintrc.js#L30) config.

**Pull request closed without merging**